### ISSUE TITLE
tfupdate -> cannot find directory 

### ISSUE THREAD

**Issue by awaismemon26** [antonbabenko/pre-commit-terraform#371]
### Describe the bug
Tfupdate hook working fine with pre-commit run command, but when git commit is performed. tfupdate fails to find directory 

### How can we reproduce it?
run git commit 

### Environment information

MacOS

* `git commit -m "test"` output:

```bash
failed to open path: stat modules/module1: no such file or directory
failed to open path: stat modules/module2: no such file or directory
```

$ uname -a
Darwin MBP.local 21.4.0 Darwin Kernel Version 21.4.0: Fri Mar 18 00:45:05 PDT 2022; root:xnu-8020.101.4~15/RELEASE_X86_64 x86_64
```
-->

* Tools availability and versions:
```bash
pre-commit : `v1.68.0`
python: `3.8.6`
tfupdate: `0.6.5`
```

* `.pre-commit-config.yaml`:

<details><summary>file content</summary>

```bash
- repo: https://github.com/antonbabenko/pre-commit-terraform
    rev: v1.68.0
    hooks:
      - id: tfupdate
        name: Update Google provider to specified version 
        args:
          - --args=provider google
          - --args=--version 4.18.0
      - id: tfupdate
        name: Update Terraform provider to specified version
        args:
          - --args=terraform
          - --args=--version 1.1.8
```

</details>

**Comment by MaxymVlasov** [comment]
Bug confirmed.

* `pre-commit run` not work
* `pre-commit run -a` works fine.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 1.70.1 :tada:
